test/test_distributed.py

```diff
@@ -275,36 +275,36 @@ def test_constantone(tmp_path, num_processes, dims, nsources, ntargets):
 
     from boxtree.tools import run_mpi
     run_mpi(__file__, num_processes, {
-        "PYTEST": "constantone",
-        "dims": dims,
-        "nsources": nsources,
-        "ntargets": ntargets,
         "OMP_NUM_THREADS": 1,
-        "tmp_cache_basedir": tmp_path / "boxtree_distributed_test",
-        "communicate_mpoles_via_allreduce": False
+        "_BOXTREE_TEST_NAME": "constantone",
+        "_BOXTREE_TEST_DIMS": dims,
+        "_BOXTREE_TEST_NSOURCES": nsources,
+        "_BOXTREE_TEST_NTARGETS": ntargets,
+        "_BOXTREE_TEST_TMP_CACHE_BASEDIR": tmp_path / "boxtree_distributed_test",
+        "_BOXTREE_TEST_MPOLES_ALLREDUCE": False
```

Comment:
Those SIM rules have a bunch of interesting demands
https://docs.astral.sh/ruff/rules/uncapitalized-environment-variables/